chance/p5js/colorfulcircles/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Malte Völkner
+author-url: https://github.com/MalteVoelkner
+title: Colorful circles
+tags: [P5.js, chance, Law of Chance]
```

Comment:
there still is no definition for the Law of chance. If I merge this this sketch won't appear under a tag like this. Do you want to write that definition?